app/components/primer/button_component.rb

```diff
@@ -111,5 +119,16 @@ def initialize(
     def link?
       @scheme == LINK_SCHEME
     end
+
+    def trimmed_content
```

Comment:
I forget, did you file a ViewComponent issue for this? It feels like a framework feature 😄